stix/idl/io/stx_map2fits.pro

```diff
@@ -0,0 +1,285 @@
+    fxaddpar, header, 'HEEY_OBS', sxpar(this_header,'HEEY_OBS'), '[m] s/c Heliocentric Earth Ecliptic Y'
+    fxaddpar, header, 'HEEZ_OBS', sxpar(this_header,'HEEZ_OBS'), '[m] s/c Heliocentric Earth Ecliptic Z'
+    fxaddpar, header, 'HCIX_OBS', sxpar(this_header,'HCIX_OBS'), '[m] s/c Heliocentric Inertial X'
+    fxaddpar, header, 'HCIY_OBS', sxpar(this_header,'HCIY_OBS'), '[m] s/c Heliocentric Inertial Y'
+    fxaddpar, header, 'HCIZ_OBS', sxpar(this_header,'HCIZ_OBS'), '[m] s/c Heliocentric Inertial Z'
+    fxaddpar, header, 'HCIX_VOB', sxpar(this_header,'HCIX_VOB'), '[m/s] s/c Heliocentric Inertial X Velocity'
+    fxaddpar, header, 'HCIY_VOB', sxpar(this_header,'HCIY_VOB'), '[m/s] s/c Heliocentric Inertial Y Velocity'
+    fxaddpar, header, 'HCIZ_VOB', sxpar(this_header,'HCIZ_VOB'), '[m/s] s/c Heliocentric Inertial Z Velocity'
+    fxaddpar, header, 'HAEX_OBS', sxpar(this_header,'HAEX_OBS'), '[m] s/c Heliocentric Aries Ecliptic X'
+    fxaddpar, header, 'HAEZ_OBS', sxpar(this_header,'HAEY_OBS'), '[m] s/c Heliocentric Aries Ecliptic Y'
+    fxaddpar, header, 'HAEZ_OBS', sxpar(this_header,'HAEY_OBS'), '[m] s/c Heliocentric Aries Ecliptic Z'
+    fxaddpar, header, 'HEQX_OBS', sxpar(this_header,'HEQX_OBS'), '[m] s/c Heliocentric Earth Equatorial X'
+    fxaddpar, header, 'HEQY_OBS', sxpar(this_header,'HEQY_OBS'), '[m] s/c Heliocentric Earth Equatorial Y'
+    fxaddpar, header, 'HEQZ_OBS', sxpar(this_header,'HEQZ_OBS'), '[m] s/c Heliocentric Earth Equatorial Z'
+    fxaddpar, header, 'GSEX_OBS', sxpar(this_header,'GSEX_OBS'), '[m] s/c Geocentric Solar Ecliptic X'
+    fxaddpar, header, 'GSEY_OBS', sxpar(this_header,'GSEY_OBS'), '[m] s/c Geocentric Solar Ecliptic Y'
+    fxaddpar, header, 'GSEZ_OBS', sxpar(this_header,'GSEZ_OBS'), '[m] s/c Geocentric Solar Ecliptic Z'
+    fxaddpar, header, 'OBS_VR  ', sxpar(this_header,'OBS_VR  '), '[m/s] Radial velocity s/c'
+    fxaddpar, header, 'EAR_TDEL', sxpar(this_header,'EAR_TDEL'), '[s] Time(Sun to Earth) - Time(Sun to S/C)'
+    fxaddpar, header, 'SUN_TIME', sxpar(this_header,'SUN_TIME'), '[s] Time(Sun to s/c)'
+    earth_time = anytim(time_range[0]) + anytim(sxpar(this_header,'EAR_TDEL'))
+    fxaddpar, header, 'DATE_EAR', anytim(earth_time,/ccsds), 'Start time vis interval, corrected to Earth'
+    sun_time = anytim(time_range[0]) - anytim(sxpar(this_header,'SUN_TIME'))
+    fxaddpar, header, 'DATE_SUN', anytim(sun_time,/ccsds), 'Start time vis interval, corrected to Sun'
+    
+    ; Entries that depend from the map
+    fxaddpar,header,'CROTA1',map[i].roll_angle,'[deg] Roll angle'
+    fxaddpar,header,'CROTA2',map[i].roll_angle,'[deg] Roll angle'
+    fxaddpar,header,'CUNIT1','arcsec  ','units along axis 1'
+    fxaddpar,header,'CUNIT2','arcsec  ','units along axis 2'
+    fxaddpar,header,'CROTACN1',map[i].roll_center[0],'[arcsec] Rotation x center'
+    fxaddpar,header,'CROTACN1',map[i].roll_center[1],'[arcsec] Rotation y center'
```

Comment:
Don't think these are fits4 standard should be PCi_j or the other one

Reply:
Where can I get these info as well as the values of the PCi_j? I'm not sure about their meaning and values.